editor.rkt

```diff
@@ -76,7 +78,7 @@
 
     (define/public (copy)
       (define new-editor (new lsp-editor%))
-      (send new-editor set-core! (send core copy-self))
+      (send new-editor insert (send this get-text) 0)
```

Comment:
Could you explain this changes? Is that means all we need copy text to `new-editor`?

Reply:
Yes. We only need to copy text to the new editor.

All of the public methods of `lsp-editor%` except the indent method are simply insert/delete/change/query text strings.
It does not require additional states.

The private states that `core` contains are always the default. So we just ignore them.

Reply:
Ok, your explanation can be a comment here, then this part is ok